taffy/main.go

```diff
@@ -54,6 +55,20 @@ Options:
 		"taffy_job": os.Getenv("FLYNN_JOB_ID"),
 	}
 
+	if os.Getenv("HOME") == "" || os.Getenv("HOME") == "/" {
+		os.Setenv("HOME", "/root/")
+	}
+
+	if os.Getenv("SSH_CLIENT_KEY") != "" {
+		os.Mkdir(os.Getenv("HOME")+"/.ssh/", 0700)
+		ioutil.WriteFile(os.Getenv("HOME")+"/.ssh/id_rsa", []byte(os.Getenv("SSH_CLIENT_KEY")), 0600)
+	}
+
+	if os.Getenv("SSH_CLIENT_HOSTS") != "" {
+		os.Mkdir(os.Getenv("HOME")+"/.ssh/", 0700)
+		ioutil.WriteFile(os.Getenv("HOME")+"/.ssh/known_hosts", []byte(os.Getenv("SSH_CLIENT_HOSTS")), 0600)
+	}
```

Comment:
This change is necessary because it is the `taffy` process which clones the private repo, not the receiver.